### ISSUE TITLE
Ice Box review

### ISSUE THREAD

**Issue by shmonks** [hackforla/peopledepot#386]
### Overview

We need to review issues in the Ice Box, and move any issues with released dependencies into the Prioritized Backlog.

### Action Items

- [ ] Search Project Board for `status:Ice Box` and `label:dependency` (or click on link below)
- [ ] Follow the links to the dependencies in each issue
- [ ] Dependencies released? Move the issue from the Ice Box to New Issue Review column
- [ ] Remove `dependency` label and add `Ready for...` label
- [ ] Create a comment below with today's date, listing the issues moved
- [ ] Repeat weekly

### Resources/Instructions

- [PD Ice Box issues with 'dependency' label](https://github.com/orgs/hackforla/projects/61/views/1?filterQuery=status%3A%22%F0%9F%A7%8AIce+Box%22+label%3Adependency)

**Comment by shmonks** [comment]
- #205 moved to Prioritized Backlog

**Comment by shmonks** [comment]
### 2024-10-24

- No Ice Box issues with ALL dependencies released

**Comment by shmonks** [comment]
### 2024-11-07
- No Ice Box issues with all dependencies released - but #191 in the Prioritized Backlog would close #189

**Comment by shmonks** [comment]
## 2024-11-14
Moved to Prioritized Backlog:
- #282 
Dependency to be released?:
- #290 (see https://github.com/hackforla/peopledepot/issues/289#issuecomment-2278755883 ?)

**Comment by shmonks** [comment]
### 2024-11-21

No Ice Box issues with ALL dependencies released - though once the final stages of #153 are completed, we can release the dependency on #154 (among other Design issues in the Ice Box) and move that into the Prioritized Backlog

**Comment by shmonks** [comment]
### 2025-01-30

Thorough audit of Ice Box finished. 

The following issues were moved to Prioritized Backlog:
- #37 because no longer dependent on #181 
- #61 
- #75 
- #219 
- #325 

The following issues were moved to Questions/Review:
- #67 - still dependent on #181?
- #239 - needs revision now?
- #257 - dependency needed?
- #258 - dependency needed?
- #288 
- #384 - ready for Prioritized Backlog?

The following issues were moved to the Ice Box:
- #326 
- #327 
- #328 
- #329 
- #330

**Comment by shmonks** [comment]
### 2025-02-13

No Ice Box issues with all dependencies released, though likely closure of deployment-related issues may change that

**Comment by shmonks** [comment]
### 2025-02-20

No Ice Box issues with all dependencies released.

**Comment by shmonks** [comment]
### 2025-02-27

No Ice Box issues with all dependencies released.

**Comment by shmonks** [comment]
### 2025-03-05

No Ice Box issues with all dependencies released, though I have moved the following issues into Questions/Review because it's possible their dependencies are no longer necessary:
- #62 
- #68 
- #72 
- #73 
- #82

**Comment by shmonks** [comment]
### 2025-03-12

No Ice Box issues with all dependencies released.

**Comment by shmonks** [comment]
### 2025-03-20
No Ice Box issues with all dependencies released.

**Comment by shmonks** [comment]
### 2025-03-27

No Ice Box issues with all dependencies released.

**Comment by shmonks** [comment]
### 2025-04-08

- #68 moved to New Issue Review, all dependencies released

**Comment by vanessaavviles** [comment]
No ice box issues with all dependencies released.

**Comment by vanessaavviles** [comment]
**2025-06-05**
No ice box issues with all dependencies released.

**Comment by vanessaavviles** [comment]
**2025-06-12**
No ice box issues with all dependencies released.

**Comment by vanessaavviles** [comment]
**2025-06-18**
No ice box issues with all dependencies released.

**Comment by vanessaavviles** [comment]
**2025-10-23**
No ice box issues with all dependencies released.
